apps/builder/app/builder/features/props-panel/controls/url.tsx

```diff
@@ -175,29 +186,104 @@ const BaseEmail = ({ prop, onChange, id }: BaseControlProps) => {
   );
 };
 
+const useSections = (page: { rootInstanceId: string }) => {
+  const store = useMemo(
+    () =>
+      computed([instancesStore, propsStore], (instances, props) => {
```

Comment:
hmm, not sure why you need computed, seems like you could just call .get() on those stores

Reply:
I call `useStore()` on the `store` created by `computed`. `computed` inside `useMemo` is a pattern we use everywhere